app/components/page-components/MainNavListItems.jsx

```diff
@@ -119,7 +119,7 @@ class MainListItems extends React.Component {
           </ListItem>
         </Link>
         <ListItem style={{display: 'inline', paddingTop: "45px"}} button={button} onClick={() => this.handleClick()}>
-          <input id="linkUpload" type="file" accept=".hcpkg" name="fileInput" onChange={this.handleUpload} ref="fileInput" style={{display:"none"}}/>
+          <input id="linkUpload" type="file" accept=".hcpkg, .json" name="fileInput" onChange={this.handleUpload} ref="fileInput" style={{display:"none"}}/>
```

Comment:
HI @Connoropolous, Should we be letting users upload .json? Because we do not have a way to figure out if the bundle is valid dna or not.
What do you think?

Reply:
Just cause they've changed the file extension to .hcpkg doesn't give us any inherent validation. It's still just a JSON file. 
In any case, there are changes likely coming soon where those files will be called (by default) `app-name.dna.json` so that will be useful and valid then

Reply:
Ok, then, for now, we move ahead by accepting both the extension and then once the switch too `.dna.json` we restrict it.